bot/exts/evergreen/fun.py

```diff
@@ -41,7 +41,7 @@ async def roll(self, ctx: Context, num_rolls: int = 1) -> None:
         elif num_rolls < 1:
             output = ":no_entry: You must roll at least once."
         for _ in range(num_rolls):
-            terning = f"terning{random.randint(1, 6)}"
+            terning = f"dice_{random.randint(1, 6)}"
```

Comment:
Maybe also rename the variable to `dice` or `die`?

Reply:
Sure; we could do that. The emojis on the server are called `dice_X`, though.

If we rename this, I think we should rename the server icons as well.